snap/hooks/configure

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
This is a workaround for snapcraft 7, correct? Did you ever get a response from the snapcraft team on this? At minimum, we should probably add a note to this script noting that it's a workaround.

Reply:
As mentioned in comments, I've reported the bug.

We are no longer affected by it because we have refactored the hooks to have a single binary executable. This reduces the build complexity (code, time) and snap size. The same improvement is being rolled out to other snaps.